busybox.yaml

```diff
@@ -67,6 +67,9 @@ pipeline:
       install -m755 busybox "${{targets.destdir}}"/bin/busybox
       install -m644 securetty "${{targets.destdir}}"/etc/securetty
 
+      mkdir -p "${{targets.destdir}}"/etc/busybox-paths.d
+      ./busybox --list-path > "${{targets.destdir}}"/etc/busybox-paths.d/busybox
```

Comment:
```suggestion
      "${{targets.destdir}}"/bin/busybox --list-path > "${{targets.destdir}}"/etc/busybox-paths.d/busybox
```
?

Reply:
I don't really think it matters which we do.  We have `busybox` in the working dir which gets installed.